common/config/rush/.pnpmfile.cjs

```diff
@@ -1,20 +1,23 @@
 function readPackage(pkg) {
   if ((pkg.name == "typedoc" || pkg.name == "@microsoft/api-extractor") && pkg.dependencies && pkg.dependencies["typescript"])
     pkg.dependencies["typescript"] = "~4.4.0";
+  // Hacky mess: For external packages to this monorepo that have peer dependencies on packages
```

Comment:
im surprised we dont need to do this for the reality-data, projects, browser-auth clients
@calebmshafer

Reply:
I think it's due to they're just type dependencies in those packages and if you look in the lock file you'd find the same issue.